orbs.rb

```diff
@@ -0,0 +1,23 @@
+require "sinatra"
+require 'sinatra/json'
+require "sinatra/reloader" if development?
+require "sequel"
+require_relative "db.rb"
+
+
+DB = Sequel.connect(adapter: 'mysql2', host: HOST, database: DBNAME, user: USERNAME, password: PASSWORD)
```

Comment:
Very extremely nitpicky, but I feel like there should only be one blank line after `require_relative` and also a blank line between the database declaration and the first route.